Add missing target dependency to pinocchio to get the right flags

--- CMakeLists.txt.orig	2019-03-05 15:28:26.000000000 +0100
+++ CMakeLists.txt	2019-03-05 15:30:03.548372096 +0100
@@ -71,6 +71,7 @@
 IF(BUILD_PYTHON_INTERFACE)
   PYTHON_ADD_MODULE(robot_utils_sot_py src/tools/robot-utils-py.cpp)
   PKG_CONFIG_USE_DEPENDENCY(robot_utils_sot_py dynamic-graph)
+  PKG_CONFIG_USE_DEPENDENCY(robot_utils_sot_py pinocchio)
   TARGET_LINK_LIBRARIES(robot_utils_sot_py ${Boost_LIBRARIES} ${PYTHON_LIBRARIES} ${LIBRARY_NAME})
   TARGET_LINK_BOOST_PYTHON(robot_utils_sot_py)
   INSTALL(TARGETS robot_utils_sot_py DESTINATION ${PYTHON_INSTALL_DIR})
